Parsers/ASimAuditEvent/Parsers/vimAuditEventVectraCloudAudit.yaml

```diff
@@ -0,0 +1,74 @@
+Parser:
+  Title: Audit Event ASIM filtering parser for VectraCloud Audit Logs Event
+  Version: '0.1'
+  LastUpdated: Mar 17 2023
+Product:
+  Name: Vectra
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Audit Event Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports filtering and normalizing VectraCloud Audit Logs Event in the Audits_Data_CL table to the ASIM Audit Event schema.
+ParserName: vimAuditEventVectraCloudAudit
+EquivalentBuiltInParser: _Im_AuditEvent_VectraCloudAudit
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+  - Name: eventresult
+    Type: string
+    Default: '*'
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: actorusername_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: operation_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: object_has_any
+    Type: dynamic
+    Default: dynamic([])
+ParserQuery: |
+  let parser = (disabled:bool = false, eventresult:string='*', starttime:datetime=datetime(null), endtime:datetime=datetime(null), actorusername_has_any:dynamic=dynamic([]),operation_has_any:dynamic=dynamic([]),object_has_any:dynamic=dynamic([]))
+  {
+    Audits_Data_CL
+    | where (isnull(starttime) or event_timestamp_t >= starttime) and (isnull(endtime) or event_timestamp_t <= endtime) and (array_length(actorusername_has_any) == 0 or tostring(toint(user_id_d)) has_any (actorusername_has_any)) or (array_length(actorusername_has_any) == 0 or username_s has_any (actorusername_has_any)) and (array_length(operation_has_any) == 0 or event_action_s has_any (operation_has_any)) and (array_length(object_has_any) == 0 or event_object_s has_any (object_has_any))
```

Comment:
Filter for disabled is missing for all the parsers

Reply:
Updated.